sdcm/cluster.py

```diff
@@ -4635,8 +4635,9 @@ def check_cluster_health(self):
             # TODO: find how to recognize, that nemesis on the node is running
             if self.nemesis_count == 1:
                 node_for_timeout = next((n for n in self.nodes if not n.running_nemesis), self.nodes[0])
+                timeout_multiplier = 2 if self.params.get('cluster_backend') == 'xcloud' else 1
```

Comment:
That's bad, or we put the timeout into configuration or into xcloud implemention

Also, why is it much slower ? It means we gave something failing again and again, we need to sort it out, not just wait longer